lib/CCreatureHandler.h

```diff
@@ -335,9 +327,6 @@ class DLL_LINKAGE CCreatureHandler : public CHandlerBase<CreatureID, Creature, C
 		h & skillLevels;
 		h & skillRequirements;
 		h & commanderLevelPremy;
-		h & allCreatures;
-		h & creaturesOfLevel;
-		BONUS_TREE_DESERIALIZATION_FIX

```

Comment:
1) please bump serialisation and min_serialization versions
2) please test multiplayer because I definitely remember that bonus_serialization_fix was one of essential things needed to pass map over network

Reply:
I will bump serialization version when all other complains will be resolved. There is no need to de serialization fix for creature handler now, because now it is not have any bonus nodes inside. All other de serialization fixes are in place.

Reply:
Serialization verson bumped. Network game seems to work on one computer with 2 clients.